### PR TITLE
feat: ul and li elements added

### LINKED ISSUE #1497: Improve Wallet species planted and token section
# Description

We can improve the **Species of trees planted** section by updating the elements. Currently the "list" is all made out of divs. We can improve this by using `ul` and `li` tags. There is also a "list" for the tokens which should also be structured as a real list.

Check [here for the code](https://github.com/Greenstand/treetracker-web-map-client/blob/beta/src/pages/wallets/%5Bwalletid%5D.js#L314-L344).

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- For design guidelines checkout our [Figma design](https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap).
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by pratham-yuxo**
# Description
ul and li tags are added to the list to improve SEO.

Fixes #1497 

## Type of change SEO

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -7,4 +7,4 @@ NEXT_PUBLIC_API_NEW=http://127.0.0.1:4010/mock
 NEXT_PUBLIC_BASE=
 NEXT_PUBLIC_CONFIG_API=
 NEXT_PUBLIC_COUNTRY_LEADER_BOARD_DISABLED=false
-NEXT_PUBLIC_SERVER_CONFIG_DISABLED=true
+NEXT_PUBLIC_SERVER_CONFIG_DISABLED=
```
Comment (RubenSmn) [inline .env.development:10]: I don't think this change should be in this PR

**Review by RubenSmn — CHANGES_REQUESTED** [review]
Nice work! I now see that I forgot to mention in the issue that there is also a `tokens` "list" which should also be structured as a list. The code is on **line 308**. For that I recommend changing the `component` property on the `Box`.

I'll update the issue accordingly!

File: .env.development (hunk 1 above)
Comment (pratham-yuxo) [reply .env.development:10]: I use this for local development , I will remove that or make a new pr.

File: .env.development [hunk 2]
```diff
@@ -1,10 +0,0 @@
-NEXT_PUBLIC_API=https://dev-k8s.treetracker.org/query
-NEXT_PUBLIC_TILE_SERVER_URL=https://{s}.treetracker.org/tiles/
-NEXT_PUBLIC_TILE_SERVER_SUBDOMAINS=dev-k8s
-NEXT_PUBLIC_TILE_SERVER_WEBMAP_API=https://dev-k8s.treetracker.org/webmap/
-NEXT_PUBLIC_IMAGE_API=https://dev-k8s.treetracker.org/images
-NEXT_PUBLIC_API_NEW=http://127.0.0.1:4010/mock
-NEXT_PUBLIC_BASE=
-NEXT_PUBLIC_CONFIG_API=
-NEXT_PUBLIC_COUNTRY_LEADER_BOARD_DISABLED=false
-NEXT_PUBLIC_SERVER_CONFIG_DISABLED=true
```
Comment (dadiorchen) [inline .env.development:10]: @RubenSmn this file is useful, we shouldn't delete it

Comment (RubenSmn) [reply .env.development:10]: Yes I know, this PR is already closed :+1: @dadiorchen

**Pull request closed without merging**